mypy/main.py

```diff
@@ -568,11 +568,7 @@ def add_invertible_flag(
     )
 
     general_group.add_argument(
-        "-O",
-        "--output",
-        metavar="FORMAT",
-        help="Set a custom output format",
-        choices=OUTPUT_CHOICES,
+        "-O", "--output", help="Set a custom output format", choices=sorted(OUTPUT_CHOICES)
```

Comment:
(Informational:) an alternate form I considered for this code was

```py
        "-O",
        "--output",
        metavar="FORMAT",
        # The metavar overrides the default of displaying the choices,
        # so we have to explicitly display them.
        help=f"Set a custom output format (choices: {set(OUTPUT_CHOICES.keys())})",
        choices=OUTPUT_CHOICES,
```

However, I ultimately decided that just letting it do its default thing was better.